hedera-mirror-rest/__tests__/specs/accounts/{id}/rewards/all-params.json

```
@@ -0,0 +1,52 @@
+{
+  "description": "Account staking reward payouts api call with all applicable params",
+  "setup": {
+    "entities": [
+      {
+        "num": 1001
+      },
+      {
+        "num": 800
+      }
+    ],
+    "stakingRewardTransfers": [
+      {
+        "accountId": "0.0.1001",
```

use snake case since that's the naming convention of the db column names

```suggestion
        "account_id": "0.0.1001",
```

---

Done.